source/MRMesh/MRChangeVertsColorMapAction.h

```diff
@@ -1,6 +1,6 @@
 #pragma once
 #include "MRHistoryAction.h"
-#include "MRColor.h"
+#include "MRVisualObject.h" //Color and DIRTY_VERTS_COLORMAP
```

Comment:
accident?

Reply:
For some reason this PR does not build MR bindings because of this